### DISCUSSION TITLE
use github

### DISCUSSION THREAD

**Discussion by philoche63** [community/community#118530] [Programming Help] [answered]
### Select Topic Area

Question

### Body

Hello everyone,
I'm new to python and don't know how to use github.
I want to start a project and I am strongly recommended to use github (https://github.com/philoche63/rugbydata). How should I proceed now? Thank you in advance for your help. Have a nice afternoon

**Comment by joinemm** [comment]
You will need to learn `git` and version control in general before you can use github with it: https://git-scm.com/book/en/v2

**Reply by devashish2024** [reply]
You don't need to learn git, for beginning you can upload files directlly if not big

**Reply by philoche63** [reply, marked as answer]
thank you very much for taking the time to answer me. I just downloaded this book. Have a nice afternoon

**Reply by LiteBrite82** [reply]
Thanks for posting in the GitHub Community, @joinemm!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Reply by joinemm** [reply]
@LiteBrite82 I am just answering him. I think you mentioned the wrong person. And also selected the wrong answer.

**Comment by synasapmob** [comment]
i don't understand, what you want?
